### PR TITLE
Change emailInfo type to match the VOA Spec

### PR DESCRIPTION
**Pull request by markolson**
Based on the `emailInfo` info in the docs, it should have a similar structure to the `phoneInfo` data

### CODE REVIEW

File: src/server/enrollment-system.js [hunk 1]
```diff
@@ -1039,7 +1039,12 @@ function veteranToDemographicsInfo(veteran) {
           addressTypeCode: 'P',  // TODO(awong): this code is from VHA Standard Data Service (ADRDEV01) Address Type List P==Permanent. Determine if we need it.
         },
         emails: {
-          email: veteran.email,
+          email: [
+            {
+              address: veteran.email,
```
Comment (akainic) [inline src/server/enrollment-system.js]: Maybe change this to `emailAddress` so it's more clear? Also this seems to be the pattern `phone` uses.

File: src/server/enrollment-system.js [hunk 2]
```diff
@@ -1039,7 +1039,12 @@ function veteranToDemographicsInfo(veteran) {
...
+              type: '1'
```
Comment (akainic) [inline src/server/enrollment-system.js]: Did we verify this type against the VHA standard data service?

File: src/server/enrollment-system.js (hunk 1 above)
Comment (markolson) [inline src/server/enrollment-system.js]: From https://github.com/department-of-veterans-affairs/healthcare-application/blob/master/src/server/enrollment-system.js#L991-L992 (and the doc that comes from) it looks like the key is just `address`. I may be misunderstanding how this data is supposed to be built up, though.

File: src/server/enrollment-system.js (hunk 2 above)
Comment (markolson) [inline src/server/enrollment-system.js]: I've been going off of a VOA Data Elements excel doc and the comments above this method - is there a a better place to look?

File: src/server/enrollment-system.js (hunk 1 above)
Comment (akainic) [inline src/server/enrollment-system.js]: Ah no I think you're right, ignore this!

File: src/server/enrollment-system.js (hunk 2 above)
Comment (akainic) [inline src/server/enrollment-system.js]: I think those are the right things to reference. You can also access the VHA SDS directly, but I'm not sure if you'll be able to access it outside of the VA network: http://vaausesrapp80.aac.va.gov:7404/ds/List/EmailAddressType

**Comment by akainic** [comment]
LGTM! Once you rebase the branch feel free to merge.

File: src/server/enrollment-system.js (hunk 2 above)
Comment (awong-dev) [inline src/server/enrollment-system.js]: Interestingly enough, the ES system has accepted our previous submissions without this type... this is something that I might send Joshua?

Comment (markolson) [inline src/server/enrollment-system.js]: Absolutely, and I'll hold off on merging until then. A concern I have is that the validator on that side is allowing extra elements through but silently dropping them, and only raising errors on malformed elements that it has a schema for.

**Pull request merged**